### ISSUE TITLE
Start Feed Map: Fall-down Map when scrolling down

### ISSUE THREAD

**Issue by MattiasJay** [Skjutsgruppen/skjutsgruppen-reactnative#301]
- [x] fall-down map implemented in ios
- [x] fall-down map implemented in android 

I don't think we have a previous issue about this.

The idea is that the map in the start feed should "fall down" when the feed is being scrolled down. @anjhero have started to work with an animation for this, which resembles the rounded "feel" (almost like water) that is applied generally in the app.

- Loading new cards is done by scrolling the other way, just like you have done as well.

**Comment by MattiasJay** [comment]
Hi @ruchidhami @anjesh @shresthaKiran ,

thanks for the new update in the App Store today. I notice that you in earlier conversations and in the check box above have say that you have imlemented the falling down of the maps. In todays update the maps in rides, groups and start feed still don't fall down (iOS). Give me an update about this. Ping @anjhero .

**Comment by anjesh** [comment]
Yes it was working in ios but we didn't deploy the changes because it conflicts with the pull-down-refresh feature. What do you think we should do? @MattiasJay 

Also we don't think we will be able to do the animation part as well.

In Android, it's still not possible after having spent almost 2 weeks trying to get this working.

**Comment by MattiasJay** [comment]
Hi again!

Suggestion: IF scrolled down THEN refresh when going back to the feed/group/ride

@anjesh  @ruchidhami

**Comment by saugatmaharjan** [comment]
Hi @MattiasJay,
Sorry, but your suggestion is bit unclear. Can you please explain the idea little more specifically?

**Comment by anjesh** [comment]
@MattiasJay waiting for your clarification?

**Comment by MattiasJay** [comment]
Like this:

1) Scroll down the map from top of start feed and/or group or ride
2) When map is scrolled up again THEN automatic reload of page is triggered.

Makes sense @saugatmaharjan @anjesh ?

**Comment by ruchidhami** [comment]
Done for both IOS and Android .

**Comment by MattiasJay** [comment]
I'm looking at this on Test Flight, version 1.3.0, iPhone.
- Start feed implementation of falling map destroys design.
- ...also completly wrong design after map has fallen down on start feed (there's a menu glued to bottom)
- Falling map in rides does not work  (tries do drag down, nothing happens)
- Falling map in group does not work (tries do drag down, nothing happens)

Important; do **not** publish version 1.3.0 to App Store since it destroys the start feed design. @ruchidhami @anjesh @anjhero
